packages/volto/locales/de/LC_MESSAGES/volto.po

```diff
@@ -635,7 +635,7 @@ msgstr "Änderungen gespeichert."
 #. Default: "Check this box to customize the title, description, or image of the target content item for this teaser. Leave it unchecked to show updates to the target content item if it is edited later."
 #: components/manage/Blocks/Teaser/schema
 msgid "Check this box to customize the title, description, or image of the target content item for this teaser. Leave it unchecked to show updates to the target content item if it is edited later."
-msgstr ""
+msgstr "Aktivieren Sie dieses Kontrollkästchen, um den Titel, die Beschreibung oder das Bild des Zielinhalts für diesen Teaser anzupassen. Lassen Sie es deaktiviert, um Aktualisierungen des Zielinhalts anzuzeigen, falls dieser später bearbeitet wird"
```

Comment:
@Tishasoumya-02 there should be a dot at the end of this sentence. Could be that I forgot that in my spec. Can you please add it? Please also replace "Bild" with "Vorschaubild" (again, sorry for changing the specs).